plugins/usertiming.js

```diff
@@ -87,21 +93,49 @@
 		options: {"from": 0, "window": BOOMR.window},
 
 		/*
+		 * Gets the user timings, filters out those that have already been sent
```

Comment:
Ideally has a `.` at the end of the sentence so it doesn't run when JSDoc is compiled.